src/translations/ja.translation

```diff
@@ -81,7 +81,7 @@
         <name>DefaultHouseholdName</name>
     </customLabels>
     <customLabels>
-        <label><!-- DefaultHouseholdTransactionCurrency --></label>
+        <label>￥</label>
```

Comment:
Never mind, this actually makes sense, it's the Yen sign that replaces the Dollar sign.